docs/docs-content/clusters/data-center/nutanix/install-pcg.md

```diff
@@ -0,0 +1,171 @@
+  cluster.cluster.x-k8s.io/carolina-cluster created
+  machinedeployment.cluster.x-k8s.io/carolina-cluster-wmd created
+  machinehealthcheck.cluster.x-k8s.io/carolina-cluster-mhc created
+  kubeadmcontrolplane.controlplane.cluster.x-k8s.io/carolina-cluster-kcp created
+  nutanixcluster.infrastructure.cluster.x-k8s.io/carolina-cluster created
+  nutanixmachinetemplate.infrastructure.cluster.x-k8s.io/carolina-cluster-mt-0 created
+```
+
+
+### Install CNI on Workload Cluster
+
+6. Deploy a Container Network Interface (CNI) pod network in the workload cluster to enable pod-to-pod communication. For more information, refer to [Deploy a CNI solution](https://cluster-api.sigs.k8s.io/user/quick-start.html#deploy-a-cni-solution) in the Nutanix [Quick Start](https://cluster-api.sigs.k8s.io/user/quick-start.htm) reference.
+
+```bash
+  clusterctl get kubeconfig $TEST_CLUSTER_NAME > $TEST_CLUSTER_NAME.kubeconfig -n $TEST_NAMESPACE
+```
+
+7. To verify the CNI pod network, issue the following command. 
+
+```bash
+  kubectl --kubeconfig=./<cluster_name>-cluster.kubeconfig get nodes
+```
+
+Output
+
+```bash hideClipBoard
+NAME                           STATUS   ROLES           AGE   VERSION
+test-cluster-kcp-qhb5h         Ready    control-plane   26h   v1.26.7
+test-cluster-wmd-gdjps-gx267   Ready    <none>          26h   v1.26.7
+```
+
+## Validate
+
+Use the steps below to verify your VMS are created.
+
+1. In the Nutanix Prism Element web console, navigate to **VM**. 
+
+2. In the **Table** tab, verify the VMs you created are listed.
+
+
+## Install PCG
+
+Use the following steps to install the PCG in your Kubernetes cluster.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu**, select **Tenant Settings**.
+
+3. Next, on the **Tenant Settings Menu**, select **Private Cloud Gateways** and click on **Add New Private Cloud Gateway**.
+
+4. Select **Self Hosted** in the next window that Palette displays.
+
+5. Provide a name for the PCG and use the **drop-down Menu** to select Nutanix as the cloud type. Click **Confirm** to continue. Palette displays the Private Cloud Gateway Overview page. 
+
+6. To install the Palette agent, copy the kubectl commands from the slide-out panel and execute them against your Kubernetes cluster.
+
+7. Close the slide-out panel when you have copied both the commands. The **Cluster Status** field on the PCG Overview page displays **Pending** while the PCG is deploying. Deployment is complete when the **Cluster Status** field displays **Running**.  
+
+
+## Validate
+
+When deployed, the PCG registers itself with Palette. Use the steps below to verify if the PCG is registered.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+
+2. Navigate to the **left Main Menu** and select **Tenant Settings**.
+
+
+3. Next, on the **Tenant Settings Menu**, select **Private Cloud Gateways**.
+
+
+4. Locate the PCG and verify it is installed and in the **Running** state. 
+
+
```

Comment:
What does the user do with the kind clusters after this point? Can we clean it up? If so, let's add the steps to clean up.

Reply:
Added that they can delete it.